pkgs/development/python-modules/x-transformers/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pytest-runner,
+  setuptools,
+  wheel,
+  einops,
+  torch,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "x-transformers";
+  version = "1.32.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "x_transformers";
+    inherit version;
+    hash = "sha256-NUIlJ+N2/6kz3rI0oc7bfYVWnMkZhvfljQ6zSwBPkWo=";
+  };
+
+  build-system = [ setuptools ];
+  nativeBuildInputs = [
+    wheel
+    pytest-runner
+  ];
```

Comment:
These deps seems weird, I believe you can substitute out the `pytest-runner` of `setup_requires` at it's not a real dependency?

Reply:
No it's needed. here is the output after removing that dependency 
![image](https://github.com/user-attachments/assets/995f4ec2-8a8b-4972-b994-c664ae574dd0)

Reply:
Please apply a patch and remove such a dependency.

> The pythonRelaxDepsHook has no effect on build time dependencies, such as those specified in build-system. If a package requires incompatible build time dependencies, they should be removed in postPatch through substituteInPlace or similar.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook

Reply:
@natsukium it's done!